packages/appimage/AppImageBuilder.yml

```diff
@@ -0,0 +1,106 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+version: 1
+
+script:
+  # Download, configure and make
+#  - git clone https://github.com/ygrek/mldonkey.git
+#  - cd mldonkey
+#  - ./configure --enable-batch --enable-upnp-natpmp --disable-directconnect --disable-fasttrack --disable-gnutella --disable-gnutella2 --enable-gui=newgui2 -->
+#  - make
+  # Remove any previous build
+  - rm -rf AppDir | true
```

Comment:
```suggestion
  - rm -rf AppDir
```

Reply:
I think that this line was from the reference example and the rationale for the forced success return was that if AppDir doesn't exist the build will fail.

Reply:
-f option to rm means it will not fail

Reply:
Fair enough, that's correct.
We could try to remove it despite being shown in the example recipes by the author for some reason.
Don't know if that could have any consequences.